src/core/core_version.h.in

```diff
@@ -7,9 +7,9 @@
 namespace jumanpp {
 namespace core {
 
-constexpr const StringPiece JPP_VERSION_STRING { "@JUMANPP_FULL_VERSION@" };
+static const StringPiece JPP_VERSION_STRING { "@JUMANPP_FULL_VERSION@" };
```

Comment:
I'm curious what is error here?

Reply:
This was a fun one.

A string literal constructor of a StringPiece (a one that is for const char &array[N]) was giving absurd results when used as a constexpr on MSVC. I thought that the easiest way was to get rid of it on that platform.

Reply:
Now that's really sound like a MSVC bug...
It is just doesn't look nice since static makes it not thread safe

Reply:
Constants are always thread safe. I agree that constexpr would be better, but static const is also ok.

And you are mistaken about the meaning of static here: it basically means that each compilation unit gets its own copy of the thing and can treat it as a local (non-shared) object. static has too much meanings in C++.
Usually that makes it possible to the compiler to inline things and do some other optimizations.

Reply:
Ah my bad, i forgot that we're talking about header here.